charts/amoro/values.yaml

```diff
@@ -254,7 +254,7 @@ optimizer:
     properties:
       namespace: "default"
       kube-config-path: "～/.kube/config"
-      image: "arctic163/amoro:0.6"
+      image: "apache/amoro:0.6"
```

Comment:
We shouldn't hardcore a version in the default values.yml.  I will fix this in the other PR.

Reply:
OK